docs/readme.md

```diff
@@ -0,0 +1,42 @@
+## How to use DiffusionBee
+
+This documentation is written for version 0.3.0. Parts of it may be unapplicable for other versions.
```

Comment:
Versioning is not required as it is implicit that documentation should work for the latest release version. Versioning is also kept within the git history.

Reply:
In my experience documentation often lags behind the code, which is why I put a version reference in the docs. Leaning towards keeping it that way, but I'm happy to listen to more arguments.